lib/screens/home.dart

```diff
@@ -118,7 +118,7 @@ class _HomePageState extends State<HomePage> {
           } else if (auth == "firebase") {
             if (firebaseFlag) {
               var user = await _auth.currentUser();
-              if (user != null) {
+              if (user != null && user.isEmailVerified) {
```

Comment:
Always start from master as a reference, it looks like you have started from the last changes made in your local.